ush/global_cycle.sh

```diff
@@ -55,46 +53,46 @@
 #     CYCLEXEC      Program executable.
 #                   Defaults to ${EXECgfs}/global_cycle$XC
 #     FNGLAC        Input glacier climatology GRIB file.
-#                   Defaults to ${FIXam}/global_glacier.2x2.grb
+#                   Defaults to ${FIXgfs}/am/global_glacier.2x2.grb
 #     FNMXIC        Input maximum sea ice climatology GRIB file.
-#                   Defaults to ${FIXam}/global_maxice.2x2.grb
+#                   Defaults to ${FIXgfs}/am/global_maxice.2x2.grb
 #     FNTSFC        Input SST climatology GRIB file.
-#                   Defaults to ${FIXam}/RTGSST.1982.2012.monthly.clim.grb
+#                   Defaults to ${FIXgfs}/am/RTGSST.1982.2012.monthly.clim.grb
 #     FNSALC        Input Salinity climatology netcdf file.
-#                   Defaults to ${FIXam}/global_salclm.t1534.3072.1536.nc
+#                   Defaults to ${FIXgfs}/am/global_salclm.t1534.3072.1536.nc
 #     FNSNOC        Input snow climatology GRIB file.
-#                   Defaults to ${FIXam}/global_snoclim.1.875.grb
+#                   Defaults to ${FIXgfs}/am/global_snoclim.1.875.grb
 #     FNZORC        Input roughness climatology.
 #                   Defaults to igbp vegetation type-based lookup table
 #                   FNVETC must be set to igbp file:
 #                   ${FIXam}/global_vegtype.igbp.t$JCAP_CASE.$LONB_CASE.$LATB_CASE.rg.grb
 #     FNALBC        Input 4-component albedo climatology GRIB file.
-#                   defaults to ${FIXam}/global_snowfree_albedo.bosu.t$JCAP_CASE.$LONB_CASE.$LATB_CASE.rg.grb
+#                   defaults to ${FIX_SFC}/${CASE}.mx${OCNRES}.snowfree_albedo.tileX.nc
 #     FNALBC2       Input 'facsf' and 'facwf' albedo climatology GRIB file.
-#                   Defaults to ${FIXam}/global_albedo4.1x1.grb
+#                   Defaults to ${FIX_SFC}/${CASE}.mx${OCNRES}.facsf.tileX.nc
```

Comment:
I've also done away with `FIX_SFC` and `FIXsfc` on the global workflow side. I replaced it with `${FIXgfs}/orog/${CASE}/sfc`. So line 72 would be:
```
${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.facsf.tileX.nc
```

Reply:
Done at 97b765e.